### PR TITLE
Identify redundant EMP500 records and generate queries:run delete commands

### PR DESCRIPTION
**Pull request by aclum**
adding code to find workflow records that have the same was_informed_by values, find their data dependencies, and make queries: run deletion requests for duplicates. See https://github.com/microbiomedata/nmdc_automation/issues/162. This cleans up extra records for EMP500

### CODE REVIEW

File: nmdc_schema/identify_workflow_duplicates_emp500.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
+import json
+import pymongo
+import os
+from pymongo import MongoClient
+from pymongo.errors import ConnectionFailure
+from dotenv import load_dotenv
+import urllib
+
+# this code deletes multiple workflow records from EMP500 which have different base IDs with a .1 version.
```
Comment (mbthornton-lbl) [inline nmdc_schema/identify_workflow_duplicates_emp500.py:9]: This should be more accurate - this is generating a .json data file that can be used to delete these records by submitting it to the appropriate endpoint

File: nmdc_schema/identify_workflow_duplicates_emp500.py [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+    )
+    for rbt_dup in rbt_duplicate_cursor:
+        rbt_delete.append(rbt_dup["id"])
+        data_objects_to_delete(rbt_dup["has_output"])
+
+
+for doc in test_list:
+    omics_id = doc["_id"]
+    records = pick_records(omics_id)
+    if len(list(records)) == 1:
```
Comment (mbthornton-lbl) [inline nmdc_schema/identify_workflow_duplicates_emp500.py:152]: what should happen if len(list(records)) != 1?

**Review by mbthornton-lbl — CHANGES_REQUESTED** [review]

File: nmdc_schema/identify_workflow_duplicates_emp500.py (hunk 1 above)
Comment (aclum) [reply nmdc_schema/identify_workflow_duplicates_emp500.py]: updated

File: nmdc_schema/identify_workflow_duplicates_emp500.py (hunk 2 above)
Comment (aclum) [reply nmdc_schema/identify_workflow_duplicates_emp500.py:153]: added else print statement

**Review by mbthornton-lbl — APPROVED** [review]

**Pull request merged**